### PR TITLE
smoothly scroll projector

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/core/static/js/core/projector.js [hunk 1]
```diff
@@ -169,14 +192,17 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
...
+                setScroll($scope.projector.scroll);
             } else {
                 // Blank projector on error
                 $scope.elements = [];
                 $scope.projector = {
-                    scroll: 0,
                     scale: 0,
                     blank: true
                 };
+
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/projector.js]: Please remove this line.

File: openslides/core/static/js/core/projector.js [hunk 2]
```diff
@@ -169,14 +192,17 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
                     setElements($scope.projector);
                     $scope.blank = $scope.projector.blank;
                 }
+
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/projector.js]: Please remove this line.

File: openslides/core/static/js/core/projector.js [hunk 3]
```diff
@@ -160,6 +161,28 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
             });
         };
 
+        var STEPS = 5;
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/projector.js]: Please add a short comment or docstring about what this is doing (small scroll animation ...)

Comment (FinnStutzenstein) [inline openslides/core/static/js/core/projector.js]: Done.

File: openslides/core/static/js/core/projector.js (hunk 2 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/projector.js]: Done.

File: openslides/core/static/js/core/projector.js (hunk 1 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/projector.js]: Done.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Done and rebased.

**Comment by emanuelschuetze** [comment]
This fixed #2843

**Pull request merged**